From: Tobias Geerinckx-Rice <me@tobias.gr>
Date: Tue, 19 Feb 2019 07:46:28 +0100
Subject: [PATCH] gnu: soundconverter: Catch (and ignore) the right error.

Without this patch and GConf:

  Traceback (most recent call last):
    File "/gnu/…/bin/...soundconverter-real-real-real", line 164, in <module>
      from soundconverter.batch import cli_convert_main
    File "/gnu/…/lib/soundconverter/python/soundconverter/batch.py", line 31, in <module>
      from soundconverter.gstreamer import TagReader
    File "/gnu/…/lib/soundconverter/python/soundconverter/gstreamer.py", line 70, in <module>
      gi.require_version('GConf', '2.0')
    File "/gnu/…/lib/python3.7/site-packages/gi/__init__.py", line 130, in require_version
      raise ValueError('Namespace %s not available' % namespace)
  ValueError: Namespace GConf not available
---
diff -Naur soundconverter-3.0.1/soundconverter/gstreamer.py soundconverter-3.0.1/soundconverter/gstreamer.py
--- soundconverter-3.0.1/soundconverter/gstreamer.py	2018-11-23 20:38:46.000000000 +0100
+++ soundconverter-3.0.1/soundconverter/gstreamer.py	2019-02-19 07:42:15.767684388 +0100
@@ -88,7 +88,7 @@
             profile = description, extension, pipeline
             audio_profiles_list.append(profile)
             audio_profiles_dict[description] = profile
-except ImportError:
+except ValueError:
     pass
 
 required_elements = ('decodebin', 'fakesink', 'audioconvert', 'typefind', 'audiorate')
